.github/workflows/ci.yml

```diff
@@ -2,24 +2,17 @@ name: Continuous Integration
 on:
   push:
     paths-ignore:
-      - "docs/**"
       - "*.md"
   pull_request:
     paths-ignore:
-      - "docs/**"
       - "*.md"
 
-env:
-  CI: true
-  COVERALLS: 0
-
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # Maintenance and active LTS
-        node-version: [10.x, 12.x, 14.x]
```

Comment:
why deleting those versions?

Reply:
`diagnostics_channel` is only available on v15 >

Reply:
Sorry yes this commit is due to skeleton. My bad